### ISSUE TITLE
Python 3.x region_name error

### ISSUE THREAD

**Issue by satterly** [guardian/athena-cli#10]
```
bash-3.2$ athena --profile personal --db sampledb
Traceback (most recent call last):
  File "/usr/local/bin/athena", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/site-packages/athena_cli.py", line 373, in main
    shell = AthenaShell(profile, region, bucket, db=args.schema, debug=args.debug)
  File "/usr/local/lib/python3.6/site-packages/athena_cli.py", line 85, in __init__
    self.athena = Athena(profile, region, bucket, debug)
  File "/usr/local/lib/python3.6/site-packages/athena_cli.py", line 229, in __init__
    self.athena = session.client('athena')
  File "/usr/local/lib/python3.6/site-packages/boto3/session.py", line 263, in client
    aws_session_token=aws_session_token, config=config)
  File "/usr/local/lib/python3.6/site-packages/botocore/session.py", line 836, in create_client
    client_config=config, api_version=api_version)
  File "/usr/local/lib/python3.6/site-packages/botocore/client.py", line 71, in create_client
    verify, credentials, scoped_config, client_config, endpoint_bridge)
  File "/usr/local/lib/python3.6/site-packages/botocore/client.py", line 281, in _get_client_args
    verify, credentials, scoped_config, client_config, endpoint_bridge)
  File "/usr/local/lib/python3.6/site-packages/botocore/args.py", line 45, in get_client_args
    endpoint_url, is_secure, scoped_config)
  File "/usr/local/lib/python3.6/site-packages/botocore/args.py", line 111, in compute_client_args
    service_name, region_name, endpoint_url, is_secure)
  File "/usr/local/lib/python3.6/site-packages/botocore/client.py", line 354, in resolve
    service_name, region_name)
  File "/usr/local/lib/python3.6/site-packages/botocore/regions.py", line 122, in construct_endpoint
    partition, service_name, region_name)
  File "/usr/local/lib/python3.6/site-packages/botocore/regions.py", line 141, in _endpoint_for_partition
    if self._region_match(partition, region_name):
  File "/usr/local/lib/python3.6/site-packages/botocore/regions.py", line 159, in _region_match
    return re.compile(partition['regionRegex']).match(region_name)
TypeError: cannot use a string pattern on a bytes-like object
```

**Comment by satterly** [comment]
Think this is fixed by #12

**Comment by satterly** [comment]
#12 didn't fix it but pretty sure #13 did.
